tools/wpt/browser.py

```diff
@@ -17,6 +17,15 @@
 uname = platform.uname()
 
 
+def _get_fileversion(binary):
+    command = "(Get-Item '%s').VersionInfo.FileVersion" % binary
+    try:
+        return call("powershell.exe", command).strip()
+    except (subprocess.CalledProcessError, OSError):
+        self.logger.warning("Failed to call %s in PowerShell" % command)
```

Comment:
`self` isn't defined here. I think you need to pass the logger in explictly.

Reply:
Gah.